.circleci/config.yml

```diff
@@ -0,0 +1,64 @@
+version: 2.1
+
+orbs:
+  # For documentation and more usage examples, see the orb's repo and registry pages:
+  # https://github.com/rainforestapp/rainforest-orb
+  # https://circleci.com/orbs/registry/orb/rainforest-qa/rainforest
+  rainforest: rainforest-qa/rainforest@1
+  heroku: circleci/heroku@1
+
+jobs:
+  test:
+    docker:
+      - image: circleci/ruby:2.2.0-node
```

Comment:
This is copy-pasted from the old `circle.yml` but we should probably use a more modern Ruby

Reply:
We need to, rather than we should: https://app.circleci.com/pipelines/github/rainforestapp/ci-sample/3/workflows/da8e2224-e07f-400f-8456-6599343322df/jobs/40/steps